### ISSUE TITLE
Remove some intrinsics

### ISSUE THREAD

**Issue by antoyo** [rust-lang/rustc_codegen_gcc#555]
See https://github.com/rust-lang/stdarch/pull/1584#issuecomment-2273048183

**Comment by g4titanx** [comment]
has this been handled?

**Comment by antoyo** [comment]
No. There still seems to be some LLVM intrinsics that could be removed.

**Comment by GuillaumeGomez** [comment]
Part of this work is in progress where I'm trying to only use intrinsics from latest LLVM version.

**Comment by antoyo** [comment]
We also have manually added intrinsics, which is the other part of this.
